src/main/java/fact/auxservice/SqliteService.java

```diff
@@ -119,15 +114,31 @@ public int hashCode() {
                 db.beginTransaction(SqlJetTransactionMode.READ_ONLY);
                 try {
                     if(service == AuxiliaryServiceName.DRIVE_CONTROL_TRACKING_POSITION){
-                        String earlier = time.minusHours(4).toString("YYYY-MM-dd HH:mm:ss");
-                        String later = time.plusMinutes(window).toString("YYYY-MM-dd HH:mm:ss");
+
+
+                        String earlierTime=time.minusHours(4).toString();
+                        String [] dateAndTime=earlierTime.split("T");
+                        String [] timeSplitZone=dateAndTime[1].split("Z");
+                        String earlier=dateAndTime[0]+" "+timeSplitZone[0].substring(0,8);
+
+                        String laterTime=time.plusMinutes(window).toString();
+                        dateAndTime=laterTime.split("T");
+                        timeSplitZone=dateAndTime[1].split("Z");
+                        String later = dateAndTime[0]+" "+timeSplitZone[0].substring(0,8);
```

Comment:
I don't understand why we cannot use a format string here?
Not very important since this class is obsolete in my opinion anyways.